src/read/archive.rs

```diff
@@ -236,18 +322,33 @@ impl<'data> ArchiveMember<'data> {
         };
 
         Ok(ArchiveMember {
-            header,
+            header: archive::MemberHeader::SystemV(*header),
             name,
             offset: file_offset,
             size: file_size,
         })
     }
 
+    /// Parse the archive member header, name, and file data.
+    ///
+    /// This reads the extended name (if any) and adjusts the file size.
+    fn parse<R: ReadRef<'data>>(
+        data: R,
+        offset: &mut u64,
+        names: &'data [u8],
+        kind: ArchiveKind,
+    ) -> read::Result<Self> {
+        match kind {
+            ArchiveKind::AixBig => Self::parse_aixbig(data, offset, &names),
+            _ => Self::parse_systemv(data, offset, &names),
+        }
+    }
+
     /// Return the raw header.
     #[inline]
-    pub fn header(&self) -> &'data archive::Header {
-        self.header
-    }
+    pub fn header(&self) -> &archive::MemberHeader {
+        &self.header
+    }    
```

Comment:
nit: trailing whitespace

Reply:
I prefer not to make `MemberHeader` public at this time. Please change this function to return `Option<&'data archive::Header>`, and also add a new function: `pub fn aix_header(&self) -> Option<&'data archive::AixHeader>`.